### ISSUE TITLE
Pagemark / Progress UX Improvement

### ISSUE THREAD

**Issue by SenorFusion** [burtonator/polar-bookshelf#388]
I almost opened another issue like #368 because of confusion around the Progress Bar and Pagemarks. I understand what is happening now, but only because I watched the videos, am familiar with incremental reading, and reconsidered my perspective.

The issues as I see it:

1. Most first time users are not familiar with the idea of incremental reading or non-linear reading

2. Most users mental model of "progress" is linear (start to finish) therefore the Progress Bar and Pagemarks are very confusing. 

I think the default mental model for many (most?) users just opening the app will be: "However far I scrolled down the page is where the progress bar should be" 

3. The name pagemarks I think maybe also has "highlighter" connotations "I am marking this info on the page here". Example: #297 

Possible Solutions / Improvements:

A. Guided walkthrough of Pagemarks tool on first-launch
B. Rename Pagemarks something that indicated progress. "Done" "Read" "Complete" "Finished" ??
C. First pagemark auto-starts and auto-fills with the user from Top of the document until manually stopped.
D. Progress bar UX at the top of document view (and maybe Repository View?) is non-linear:

The blue highlighted areas match _where_ in the document the progress was made in addition to _amount_

![pagemarks3](https://user-images.githubusercontent.com/1929777/48027960-39ccfc00-e118-11e8-88f7-e8b221bcb61a.png)

E. Click on Progress Bar Location takes you to that Location in the document

F. Highlights / Comments are visually indicated markers in the Progress Bar that are also clickable.

G. Visual indicator in progress bar of where scroll (user view) currently is located.

Prior Art inspiration: Chrome Ctrl-F Find On Page tool / Scroll Bar

**Comment by burtonator** [comment]
Yeah. I agree with a lot of what you are saying here and will probably end up moving in this direction... Something like this (maybe almost identical) has been in the back of my mind.

And definitely need to improve user onramp...

**Comment by SenorFusion** [comment]
Thanks for all the hard work @burtonator - I rarely contribute on github, but I'm all in on Polar for the long haul. It's really a game changer for Reading / Learning / Personal Knowledge Management / Archiving.
